### ISSUE TITLE
About the Convergence of Training Loss

### ISSUE THREAD

**Issue by HowieChangchn** [TIGER-AI-Lab/VLM2Vec#26]
This is an inspiring work. I am attempting to apply VLM2VEC to a personal dataset and have observed that a smaller batch size(64) tends to converge more easily, with the loss stabilizing between 0.1 and 1.2. However, with batch sizes of 512, the loss fluctuates between 3 and 7.

I would like to ask if you have encountered similar situations during training. When the data scale and batch size increase, can the loss still converge to a relatively small value? I have tried increasing the learning rate, but it seems to have little effect.

**Comment by XMHZZ2018** [comment]
@HowieChangchn 

Thank you for your interest in our work! In our case, I think the loss can still converge to a relatively small value when the batch size and data scale are sufficiently large.

**Comment by HowieChangchn** [comment]
Thank you very much for your response! I suspect that the quality of my personal data organization is not high enough. I will further investigate the reasons why the model cannot converge.
